mars/tensor/expressions/datasource/array.py

```diff
@@ -129,7 +129,8 @@ def tensor(data, dtype=None, chunk_size=None, gpu=None, sparse=False):
         if dtype is not None and data.dtype != dtype:
             return data.astype(dtype)
         return data
-    elif isinstance(data, tuple) and all(isinstance(d, TENSOR_TYPE) for d in data):
```

Comment:
Could be `isinstance(data, (tuple, list))`

Reply:
Done.